app/src/main/java/com/example/parkingsystemjava/mvp/presenter/ParkingPresenter.java

```diff
@@ -13,7 +13,12 @@ public ParkingPresenter(ParkingContract.ModelContract model, ParkingContract.Vie
     }
 
     public void onButtonPressed() {
-        model.setParkingAvailables();
         view.showParkingsAvailable(model.getParkingAvailables());
     }
+
+    @Override
+    public void setParkingLots(int parkingLots) {
+        model.setParkingAvailables(parkingLots);
+        view.showMessageOfParkingLots(parkingLots);
```

Comment:
here you should take the information from the model
`        view.showMessageOfParkingLots(model.getParkingLots());`

Reply:
done